egs/ljspeech/tts1/conf/specaug.yaml

```diff
@@ -0,0 +1,16 @@
+process:
```

Comment:
Do we need this file for ASR inference? I thought this is a training configuration file.

Reply:
We need this file for `asr_recog.py`. The error log w/o this file is the following.

```
# asr_recog.py --config conf/ob_eval/decode_asr.yaml --ngpu 0 --backend pytorch --batchsize 0 --recog-json exp/ground_truth/sym_link_denorm.ob_eval/librispeech.transformer.ngpu4_asr.dump/char_dev/split32utt/data.1.json --result-label exp/ground_truth/sym_link_denorm.ob_eval/librispeech.transformer.ngpu4_asr.result.v2/char_dev/data.1.json --model exp/librispeech.transformer.ngpu4_asr/exp/train_960_pytorch_train_pytorch_transformer.v1_aheads8_batch-bins15000000_specaug/results/model.val5.avg.best --api v2 --rnnlm exp/librispeech.transformer.ngpu4_asr/exp/irielm.ep11.last5.avg/rnnlm.model.best 
# Started at Sun Oct 27 06:04:53 JST 2019
#
2019-10-27 06:04:55,221 (asr_recog:161) INFO: python path = /home/abelab/k_inoue/work/tool/anaconda2/bin:/home/abelab/k_inoue/work/tool/pysoliton_1.4.3.0-md1.4.1
2019-10-27 06:04:55,221 (asr_recog:166) INFO: set random seed = 1
2019-10-27 06:04:55,221 (asr_recog:174) INFO: backend = pytorch
2019-10-27 06:05:04,521 (recog:30) WARNING: experimental API for custom LMs is selected by --api v2
2019-10-27 06:05:04,523 (deterministic_utils:24) INFO: torch type check is disabled
2019-10-27 06:05:04,523 (asr_utils:407) INFO: reading a config file from exp/librispeech.transformer.ngpu4_asr/exp/train_960_pytorch_train_pytorch_transformer.v1_aheads8_batch-bins15000000_specaug/results/model.json
2019-10-27 06:05:04,526 (asr_init:141) INFO: reading model parameters from exp/librispeech.transformer.ngpu4_asr/exp/train_960_pytorch_train_pytorch_transformer.v1_aheads8_batch-bins15000000_specaug/results/model.val5.avg.best
/home/abelab/k_inoue/work_space/espnet/tools/venv/lib/python3.7/site-packages/torch/nn/_reduction.py:49: UserWarning: size_average and reduce args will be deprecated, please use reduction='none' instead.
  warnings.warn(warning.format(ret))
Traceback (most recent call last):
  File "/home/abelab/k_inoue/work_space/espnet/egs/ljspeech/tts1/../../../espnet/bin/asr_recog.py", line 207, in <module>
    main(sys.argv[1:])
  File "/home/abelab/k_inoue/work_space/espnet/egs/ljspeech/tts1/../../../espnet/bin/asr_recog.py", line 184, in main
    recog_v2(args)
  File "/work/abelab/k_inoue/espnet/espnet/asr/pytorch_backend/recog.py", line 47, in recog_v2
    preprocess_args={'train': False})
  File "/work/abelab/k_inoue/espnet/espnet/utils/io_utils.py", line 55, in __init__
    self.preprocessing = Transformation(preprocess_conf)
  File "/work/abelab/k_inoue/espnet/espnet/transform/transformation.py", line 71, in __init__
    with io.open(conffile, encoding='utf-8') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'conf/specaug.yaml'
# Accounting: time=18 threads=1
# Ended (code 1) at Sun Oct 27 06:05:11 JST 2019, elapsed time 18 seconds
```